Examples/AdvancedImageReading/AdvancedImageReading.py

```diff
@@ -15,16 +15,16 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 #
-#=========================================================================
+# =========================================================================
 
 from __future__ import print_function
```

Comment:
Possibly remove all of the `from __future__ import print_function` as we no longer support Python 2.7. Probably in all examples.